ReactWindows/ReactNative/Storage/FutureAccessList.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.AccessCache;
+
+namespace ReactNative.Storage
+{
+    /// <summary>
+    /// The reason this future access list thing was introduced was a need
+    /// to transfer a file object from RN to JS and then to another RN module.
+    /// The naive approach to transfer the file path didn't work because the
+    /// file path doesn't carry read/write permissions and a UWP app can directly
+    /// access only files inside its app folder.
+    ///
+    /// Eventually, RNW will have a more general approach to store such things
+    /// and it will support other schemes like blob. At that time this class will
+    /// likely to be rewritten.
+    /// </summary>
+    public static class FutureAccessList
+    {
+        /// <summary>
+        /// FAL URIs start with this prefix.
+        /// </summary>
+        public static string Scheme => "urn:future-access-list:";
+
+        /// <summary>
+        /// Checks the syntax of the FAL URI. It doesn't check if
+        /// this URI corresponds to an entry in the FAL.
+        /// </summary>
+        public static bool IsValidURI(string uri)
```

Comment:
`URI` is typically stylized as `Uri` in .NET, see `System.Uri`

Reply:
Fixed. Though we'll now need to update a few RN modules :)